src/pages/LandingPageEz/Illustration.jsx

```diff
@@ -0,0 +1,23 @@
+import patternTwo from './assets/bg-pattern-2.svg';
+import phonePic from './assets/Illustration-app.png';
```

Comment:
Wrong import name. vite complains about it and breaks. Should be `./assets/illustration-app.png`